proto/definitions/ibc/lightclients/parlia/v1/parlia.proto

```diff
@@ -7,6 +7,14 @@ import "gogoproto/gogo.proto";
 import "ibc/core/client/v1/client.proto";
 import "google/protobuf/duration.proto";
 
+message ForkSpec {
+  oneof height_or_timestamp {
+    uint64 height = 1;
+    uint64 timestamp = 2;
+  }
+  uint64 additional_header_item_count = 3;
```

Comment:
Since the additions to the Header specifications are relative, I want to clarify the base fork in the comments. I would appreciate if you could add this information.

Reply:
Done by [4578dc7](https://github.com/datachainlab/parlia-elc/pull/83/commits/4578dc7341bd6efe1ca974dbf697d515609e93c0)